code/rsapss/Hacl.Spec.RSAPSS.fst

```diff
@@ -351,7 +363,13 @@ val rsapss_sign_compute_sgnt:
   -> dBits:size_nat{skey_len_pre t modBits eBits dBits}
   -> skey:lbignum t (2 * blocks modBits (bits t) + blocks eBits (bits t) + blocks dBits (bits t))
   -> m:lbignum t (blocks modBits (bits t)) ->
-  tuple2 bool (lseq uint8 (blocks modBits 8))
+  Pure (tuple2 bool (lseq uint8 (blocks modBits 8)))
```

Comment:
Any specific reason for `tuple2` instead of the infix operator?

Reply:
No reason :)